pywbem/_cim_operations.py

```diff
@@ -360,23 +360,28 @@ def _validate_context(context):
 
 class WBEMConnection(object):  # pylint: disable=too-many-instance-attributes
     """
-    A client's connection to a WBEM server. This is the main class of the
-    WBEM client library API.
-
-    The connection object knows a default CIM namespace, which is used when no
-    namespace is specified on subsequent WBEM operations (that support
-    specifying namespaces). Thus, the connection object can be used as a
-    connection to multiple CIM namespaces on a WBEM server (when the namespace
-    is specified on subsequent operations), or as a connection to only the
-    default namespace (this allows omitting the namespace on subsequent
-    operations).
+    A client's connection to a WBEM server or WBEM listener. This is the main
+    class of the WBEM client library API.
+
+    This class is used for communication with WBEM servers and WBEM listeners,
+    because the communication mechanisms are very similar. However, a
+    particular object of this class will connect to only one target which is
+    either a WBEM server or a WBEM listener, but never both.
+
+    When targeting a WBEM server, the connection object knows a default CIM
+    namespace, which is used when no namespace is specified on subsequent
+    WBEM operations (that support specifying namespaces). Thus, the connection
+    object can be used as a connection to multiple CIM namespaces on a
+    WBEM server (when the namespace is specified on subsequent operations),
+    or as a connection to only the default namespace (this allows omitting the
+    namespace on subsequent operations).
 
     As usual in HTTP, there is no persistent TCP connection; the connectedness
     provided by this class is only conceptual. That is, the creation of the
-    connection object does not cause any interaction with the WBEM server, and
-    each subsequent WBEM operation performs an independent, state-less
-    HTTP/HTTPS request. Usage of the `requests` Python package causes the
-    underlying resources such as sockets to be pooled, though.
+    connection object does not cause any interaction with the WBEM server or
+    WBEM listener, and each subsequent WBEM operation performs an independent,
+    state-less HTTP/HTTPS request. Usage of the `requests` Python package
+    causes the underlying resources such as sockets to be pooled, though.
```

Comment:
minor syntax.  change though to however and put it at start of sentence

Reply:
DONE